<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpsWid</class>
 <widget class="QWidget" name="IpsWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>407</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="ipsRadio">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>195</y>
     <width>371</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>IP区域</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>230</y>
     <width>371</width>
     <height>133</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>起始IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>结束IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="ipsBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="getRadio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>79</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>IP获取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>93</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>自动获取</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>91</y>
     <width>351</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="exportRadio">
      <property name="text">
       <string>IP导入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exportBtn">
      <property name="text">
       <string>导入</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>143</y>
     <width>376</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="ipRadio">
      <property name="text">
       <string>IP地址</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ipEdit">
      <property name="text">
       <string>192.168.1.163</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ipBtn">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
